addons/packages/grafana/7.5.11/package.yaml

```diff
@@ -0,0 +1,191 @@
+                  default: ReadWriteOnce
+                storage:
+                  type: string
+                  description: The storage size for persistent volume claim.
+                  default: "2Gi"
+            secret:
+              type: object
+              description: Grafana user access related configuration.
+              properties:
+                type:
+                  type: string
+                  description: The Secret Type (io.k8s.api.core.v1.Secret.type)
+                  default: "Opaque"
+                admin_user:
+                  type: string
+                  description: Username to access Grafana.
+                  default: admin
+                admin_password:
+                  type: string
+                  description: Password to access Grafana. By default is null and grafana defaults this to "admin"
+                  default: admin
+        ingress:
+          type: object
+          description: Grafana Ingress configuration.
+          properties:
+            enabled:
+              type: boolean
+              description: Whether to enable Grafana Ingress. Note that this requires contour.
+              default: true
+            virtual_host_fqdn:
+              type: string
+              description: Hostname for accessing grafana.
+              default: "grafana.system.tanzu"
+            prefix:
+              type: string
+              description: Path prefix for Grafana.
+              default: "/"
+            servicePort:
+              type: integer
+              description: Grafana service port to proxy traffic to.
+              default: 80
+            tlsCertificate:
+              type: object
+              description: Grafana Ingress TLS certificate configuration. If skipped, a cert-manager cert will be issued and the cert-manager package must be installed on your cluster.
+              properties:
+                tls.crt:
+                  type: string
+                  description: Optional cert for ingress if you want to use your own TLS cert. A self signed cert is generated by default. Note that tls.crt is a key and not nested.
+                  default: null
+                tls.key:
+                  type: string
+                  description: Optional cert private key for ingress if you want to use your own TLS cert. Note that tls.key is a key and not nested.
+                  default: null
+                ca.crt:
+                  type: string
+                  description: Optional CA certificate. Note that ca.crt is a key and not nested.
+                  default: null
+  licenses:
+    - "Apache 2.0"
+  template:
+    spec:
+      fetch:
+        - imgpkgBundle:
+            image: projects.registry.vmware.com/tce/grafana@sha256:53d41d9ac1534fc381efa5bb181aa4cac1ec26fc77c7ffadb34550930112e193
```

Comment:
This will need to be updated after make my other 2 requested changes.

Reply:
@ggpaue can you update the SHA to `9f9cb429bc753d19883341f771af0c996fe58b92774ae15615ce9bf60956adb2`  Once that is done, I can get this merged.